app/assets/javascripts/actions/index.coffee

```diff
@@ -0,0 +1,37 @@
+Screensmart.Actions =
+  addMessage: (message) ->
+    type: 'ADD_MESSAGE'
+    message: message
+
+  setAnswer: (key, value) ->
+    (dispatch) =>
+      dispatch @_setAnswer(key, value)
+      dispatch @updateResponse()
+
+  _setAnswer: (key, value) ->
+    type: 'SET_ANSWER'
+    key: key
+    value: value
+
+  updateResponse: ->
+    (dispatch, getState) ->
+      response = getState().response
+      syncResponse(response).then (data) ->
+        dispatch Screensmart.Actions.receiveResponseUpdate(data)
+
+  setResponseLoading: ->
+    type: 'SET_RESPONSE_LOADING'
+
+  receiveResponseUpdate: (data) ->
+    type: 'RECEIVE_RESPONSE_UPDATE'
+    response: data.response
+
+syncResponse = (response) ->
```

Comment:
Het is echter niet enkel een save, het werkt volgens deze stappen:
1. De gebruiker kliks op eens of oneens
2. `syncResponse` stuurt de response (hash van answers) op naar de server
3. de server reageert met een nieuw response-object op met de nieuwe question
4. de caller van `syncResponse` reageert op success of failure door de nieuwe question te tonen

`save` lijkt me hier iets te simpel, al heb je gelijk dat `sync` ook niet helemaal omvat wat het doet. Nog een andere suggestie?